client/signup/steps/domains/index.jsx

```diff
@@ -193,6 +224,10 @@ class DomainsStep extends Component {
 		} );
 	};
 
+	isExperiment() {
+		return this.state.experiment?.variationName !== null;
```

Comment:
Minor, feel free to ignore: For readability, we could also check the experiment name here:
```suggestion
		return 'domain_step_copy_test_202201' === this.state.experiment?.experimentName && this.state.experiment?.variationName !== null;
```